content/management/configuration-api/v3.2/index.mdx

```diff
@@ -2785,6 +2785,238 @@ curl -X POST \
+      * `value` - value to match (`string`)
+      * `exact_match` - if exact match, if set to `false` a `match_object.value` will be matched as substring of `customer_url`
+  * `customer_geolocation.values` - array of objects with optional values, at least one must be set; matches when at least one subrule matches
+    * `country`, eg. "United States
+    * `country_code`, eg. "US"
+    * `region`, eg. "California"
+    * `city`, eg. "Mountain View"
+
+
+</Text>
+
+<Code>
+
+<CodeSample path={'REQUEST'}>
+
+```shell
+curl -X POST \
+  https://api.livechatinc.com/v3.2/configuration/action/add_auto_access_rules \
+  -H 'Content-Type: application/json' \
+  -H 'Authorization: Bearer <your_access_token>' \
+  -d '{
+      "description": "Chats on livechat.com from United States",
+      "access": {
+        "groups": [1]
+      },
+      "rules": {
+        "customer_domain": {
+          "values": [{
+            "value": "livechat.com",
+            "exact_match": true
+          }]
+        },
+        "customer_geolocation": {
+          "values": [{
+              "country": "United States",
+              "country_code": "US"
+            }
+        }]
+      }
+    }'
+```
+
+</CodeSample>
+
+<CodeResponse>
+
+```json
+{
+  "auto_access_rules_id": "pqi8oasdjahuakndw9nsad9na"
+}
+```
+
+</CodeResponse>
+
+</Code>
+
+</Section>
+
+<Section>
+
+<Text>
+
+### Get auto access rules config
+
+#### Specifics
+
+|                |                                                                                      |
+| -------------- | ------------------------------------------------------------------------------------ |
+| Method URL     | `https://api.livechatinc.com/v3.2/configuration/action/get_auto_access_rules_config` |
+| Required scopes| `access_rules:ro`                                                                    |
+
+</Text>
+
+<Code>
+
+<CodeSample path={'REQUEST'}>
+
+```shell
+curl -X POST \
+  https://api.livechatinc.com/v3.2/configuration/action/get_auto_access_rules_config \
+  -H 'Content-Type: application/json' \
+  -H 'Authorization: Bearer <your_access_token>' \
+  -d '{}'
+```
+
+</CodeSample>
+
+<CodeResponse>
+
+```json
+{
+     [
+        {
+            "id": "1faad6f5f1d6e8fdf27e8af9839783b7",
+            "description": "Chats from Facebook or Twitter",
+            "access": {
+                "groups": [
+                    0
+                ]
+            },
+            "rules": {
+                "chat_properties": {
+                    "facebook": {
+                        "page_id": {
+                            "values": [
+                                "63121487121"
+                            ]
+                        }
+                    },
+                    "source": {
+                        "type": {
+                            "values": [
+                                "facebook",
+                                "twitter"
+                            ]
```

Comment:
I think chat_properties should be removed from auto access rules type based on documentation

Reply:
yes, `chat_properties` are no longer supported and there are no occurrences for that in DBs on any env.